plugins/build.gradle

```diff
@@ -22,4 +27,9 @@ configure(subprojects.findAll { it.parent.path == project.path }) {
     licenseFile rootProject.file('licenses/SSPL-1.0+ELASTIC-LICENSE-2.0.txt')
     noticeFile rootProject.file('NOTICE.txt')
   }
+
+  String configurationName = project.name.startsWith('repository-') ? 'repositoryPlugins' : 'nonRepositoryPlugins'
+  parent.dependencies { DependencyHandler d ->
```

Comment:
This doesn't seem right. We're adding dependencies to these configurations here _and_ resolving them from another project, which I'm pretty sure is a no-no. We should be instead adding the plugins as _artifacts_ to these configurations for external consumptions. @breskeby that sound right?

Reply:
I swapped this our for artifacts - any better?

Reply:
Yup, that should do the trick.